TheFourFifths/consus-client
# Plus button works for unserialized models

### Summary

Clicking the plus button for unserialized models now properly increments the model and increases instock and total values.

### Checklist


### How to Review

1. go to view all models
2. Press the plus for an unserialized model.
3. Modal text and toast text should reflect that the model is unserialized (shouldn't mention adding a new item)
4. Run tests and such

### Links

- [TFF-223](https://msoese.atlassian.net/browse/TFF-223)
- TheFourFifths/consus/pull/95
